server/member/hooks.ts

```diff
@@ -0,0 +1,14 @@
+import { Member } from 'server/models';
+import { createActivityHooks } from 'server/utils/activityHooks';
+import {
+	createMemberCreatedActivityItem,
+	createMemberUpdatedActivityItem,
+	createMemberRemovedActivityItem,
+} from 'server/activityItem/queries';
+
+createActivityHooks({
+	Model: Member,
+	onModelCreated: createMemberCreatedActivityItem,
+	onModelUpdated: createMemberUpdatedActivityItem,
+	onModelDestroyed: createMemberRemovedActivityItem,
```

Comment:
Super minor nit (feel free to ignore) – having a combination of generic `createXActivityItem` and more specific `createXUpdatedActivityItem` functions introduces a decision point to either write logic in the hooks files vs. the queries files which might result in split logic in the future e.g. someone might add logic to the anonymous functions in the hooks.ts file instead of keeping it all in queries.ts.